Stripe/STPAddCardViewController.m

```diff
@@ -408,8 +408,7 @@ - (UITableViewCell *)tableView:(__unused UITableView *)tableView
         default:
             return [UITableViewCell new]; // won't be called; exists to make the static analyzer happy
     }
-    cell.backgroundColor = [UIColor clearColor];
-    cell.contentView.backgroundColor = self.theme.secondaryBackgroundColor;
+    cell.backgroundColor = self.theme.secondaryBackgroundColor;
```

Comment:
If you don't set contentView background is it clear? Do you need to set both cell background and content view background to the theme color?

Reply:
It seems like `contentView.backgroundColor` defaults to clear, but probably best to set it explicitly just it's different on other iOS versions. I'll update.